src/style.css

```
@@ -1 +1,61 @@
 /* Styles go here */
+body {
+  margin: 0;
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  justify-content: space-between;
+}
+
+.logo__image {
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  padding: 23px 0;
+  margin-left: 20px;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+a:hover {
+  color: var(--color-blue);
+}
+
+a:active,
```

```suggestion
.nav__link:hover {
  color: var(--color-blue);
}

.nav__link:active,
```
style by class instead by tag. sometimes body, html or reset some default styles can be by tags.